cms/routes/api/items.js

```diff
@@ -1,16 +1,25 @@
-var keystone = require("keystone");
+const keystone = require("keystone"),
+	Items = keystone.list("Item");
 
-var Items = keystone.list("Item");
+const log4js = require("log4js"),
+	logger = log4js.getLogger();
 
-exports.single = function(req, res) {
+exports.single = function(req, res, next) {
```

Comment:
Do we still need the next parameter?

Reply:
Probably not but not sure! The linting in #28 will help with this in the future!